src/content/notion.js

```diff
@@ -35,54 +37,63 @@ togglbutton.render(
     if (root) {
       root.parentElement.prepend(wrapper);
     } else {
-      elem.querySelector('div:nth-child(2) > div:nth-child(1) > div:nth-child(1) > div:nth-child(3)').prepend(wrapper)
+      const selector = elem.querySelector('div:nth-child(2) > div:nth-child(1) > div:nth-child(1) > div:nth-child(3)')
+      if (!selector) return;
+      selector.prepend(wrapper)
     }
   }
 );
 
-togglbutton.render(
-  '.notion-topbar-action-buttons:not(.toggl)',
-  { observe: true },
-  function (elem) {
-    if (!elem) return;
+setTimeout(() => {
```

Comment:
Why the setTimeout here? Does the debounceInterval set to 2000 does not work?

Reply:
It doesnt do ot for the first render